config.yml.example

```
@@ -0,0 +1,21 @@
+expiry: 3m
```

❓ Can we add the config version here, so that we can easily change the config layout without breaking backward compatibility in the future? 

```suggestion
version: v1
expiry: 3m
```

---

If we ever need to change the format, we could add a version field ... for now it seems like over engineering!